net/pfSense-pkg-WireGuard/files/usr/local/www/wg/vpn_wg_edit.php

```diff
@@ -0,0 +1,687 @@
+				<?=gettext("Update")?>
+			</button>
+		</nav>
+	</div>
+	</div>
+
+  </div>
+</div>
+
+<div class="panel panel-default">
+	<div class="panel-heading">
+		<h2 class="panel-title">Peer Configuration</h2>
+	</div>
+	<div id="mainarea" class="table-responsive panel-body">
+		<table id="peertable" class="table table-hover table-striped table-condensed" style="overflow-x: 'visible'">
+			<thead>
+				<tr>
+					<th><?=gettext("Peer")?></th>
+					<th><?=gettext("Description")?></th>
+					<th><?=gettext("Endpoint")?></th>
+					<th><?=gettext("Port")?></th>
+					<th><?=gettext("Public key")?></th>
+					<th style="display:none;"><?=gettext("Keepalive")?></th>
+					<th style="display:none;"><?=gettext("Allowed IPs")?></th>
+					<th style="display:none;"><?=gettext("PSK")?></th>
+					<th></th>
+
+				</tr>
+			</thead>
+
+			<tbody>
+<?php
+
+				$peer_num = 0;
+
+				if (!empty($pconfig['peers']['wgpeer'])) {
+					foreach ($pconfig['peers']['wgpeer'] as $peer) {
+						print('<tr id="peer_row_' . $peer_num . '" class="peer_group_' . $peer_num . '">');
+						print("<td>" . htmlspecialchars($peer_num) . "</td>\n");
+						print("<td>" . htmlspecialchars($peer['descr']) . "</td>\n");
+						print("<td>" . htmlspecialchars($peer['endpoint']) . "</td>\n");
+						print("<td>" . htmlspecialchars($peer['port']) . "</td>\n");
+						print("<td>" . htmlspecialchars($peer['publickey']) . "</td>\n");
+
+						// hidden columns
+						print("<td style=\"display:none;\">" . htmlspecialchars($peer['persistentkeepalive']) . "</td>\n");
+						print("<td style=\"display:none;\">" . htmlspecialchars($peer['allowedips']) . "</td>\n");
+						print("<td style=\"display:none;\">" . htmlspecialchars($peer['presharedkey']) . "</td>\n");
+						print("<td style=\"display:none;\">" . htmlspecialchars($peer['peerwgaddr']) . "</td>\n");
+?>
+						<td style="cursor: pointer;">
+							<a class="fa fa-pencil" href="#" id="editpeer_<?=$peer_num?>"title="<?=gettext("Edit peer"); ?>"></a>
+							<a class="fa fa-trash text-danger no-confirm" href="#" id="killpeer_<?=$peer_num?>" title="<?=gettext('Delete peer');?>"></a>
+						</td>
+<?php
+						print("</tr>");
+						$peer_num++;
+					}
+				}
+?>
+			</tbody>
+		</table>
+	</div>
+</div>
+
+<nav class="action-buttons">
+	<button type="submit" id="editpeer_new" class="btn btn-sm btn-success" title="<?=gettext('Add new peer')?>">
+		<i class="fa fa-plus icon-embed-btn"></i>
+		<?=gettext("Add peer")?>
+	</button>
+
+	<button type="submit" id="saveform" name="saveform" class="btn btn-sm btn-primary" value="save" title="<?=gettext('Save tunnel')?>">
+		<i class="fa fa-save icon-embed-btn"></i>
+		<?=gettext("Save")?>
+	</button>
+</nav>
+
+<?php $jpconfig = json_encode($pconfig, JSON_HEX_APOS); ?>
+<?php $jwg_config = json_encode($wgg['config'], JSON_HEX_APOS); ?>
+
+<?php $genkeywarning = gettext("Are you sure you want to overwrite keys?"); ?>

```

Comment:
Please reword this to avoid addressing the user directly. It could be `Overwrite keys?`, `Click <button name> to overwrite keys`, or similar.